README.md

```diff
@@ -7,6 +7,10 @@ This list aims to be a curated set of high quality educational resources. The av
 
 ## Beginner
 
+#### [CS50's Introduction to Computer Science](https://www.edx.org/learn/computer-science/harvard-university-cs50-s-introduction-to-computer-science/)
+**Free** A beginner friendly Computer Science & Programming course straight from Havards available on Edx.
+*(Scratch, C, Python, SQL, JavaScript, CSS, HTML)*
+
```

Comment:
we aleady list edX further down in the list, maybe this doesn't make sense?

Reply:
I see thank you. I didn't noticed. I had forked this repo a few years ago. And when I did workflow got created. I'm not used to GitHub so apologize for inconvenience I caused you.

Reply:
no worries, no inconvenience :) 

happy to guide and help out while you get accustomed to the PR workflow

but let's close this one for now, and if you see any other improvements, you can try to open a new PR